stubs/docker/docker/api/container.pyi

```diff
@@ -66,18 +69,33 @@ class ContainerApiMixin:
     def get_archive(self, container, path, chunk_size=2097152, encode_stream: bool = False): ...
     def inspect_container(self, container): ...
     def kill(self, container, signal: Incomplete | None = None) -> None: ...
+    @overload
     def logs(
         self,
         container,
         stdout: bool = True,
         stderr: bool = True,
-        stream: bool = False,
+        stream: Literal[False] = False,
         timestamps: bool = False,
         tail: str = "all",
         since: Incomplete | None = None,
         follow: Incomplete | None = None,
         until: Incomplete | None = None,
-    ): ...
+    ) -> bytes: ...
+    @overload
+    def logs(
+        self,
+        container,
+        stdout: bool = True,
+        stderr: bool = True,
+        *,
```

Comment:
This `*` in particular I wasn't sure about. Maybe instead I should remove the `= True` on `stdout` and `stderr`?